library/src/webapp/js/portal/portal.sidebar.sites.js

```diff
@@ -20,14 +20,16 @@ class SitesSidebar {
 
         e.preventDefault();
 
-        fetch(`/direct/userPrefs/updateKey/${portal.user.id}/sakai:portal:sitenav?selectedPage=${e.target.dataset.pageId}`, { method: "PUT" })
+        const el = e.currentTarget;
+
+        fetch(`/direct/userPrefs/updateKey/${portal.user.id}/sakai:portal:sitenav?selectedPage=${el.dataset.pageId}`, { method: "PUT" })
```

Comment:
are we positive we have a real user id here? i see lots of undefined user fetches in my network console these days